infra/README.md

```diff
@@ -0,0 +1,31 @@
+# Continuous delivery pipelines
+
+This package uses the [AWS Cloud Development Kit (AWS)](https://github.com/awslabs/aws-cdk) to model AWS CodePipeline pipelines and to provision them with AWS CloudFormation.
+
+* pipeline.ts: Builds and publishes the base Docker image for the backend API service
```

Comment:
is this a copy/pasta typo? "backend API service"

Reply:
dang caught red handed. will fix